### PR TITLE
do not apply inline styles in the middle of the word

### PR DESCRIPTION
**Pull request by andrewfan**
fixes for https://github.com/withspectrum/draft-js-markdown-plugin/issues/98

### CODE REVIEW

**Comment by brianlovin** [comment]
Oh awesome! @mxstbr this might solve a lot of problems with links containing underscores, too :)

File: src/modifiers/handleInlineStyle.js [hunk 1]
```diff
@@ -14,6 +14,10 @@ const handleChange = (editorState, line, whitelist) => {
         do {
           matchArr = re.exec(line);
           if (matchArr) {
+            if (matchArr[0][0] === " ") {
+              matchArr[0] = matchArr[0].trimLeft();
```
Comment (mxstbr) [inline src/modifiers/handleInlineStyle.js]: `trimLeft` (which is an alias of `trimStart`) has horrendous browser support, can we do this manually?

File: src/modifiers/handleInlineStyle.js [hunk 2]
```diff
@@ -14,6 +14,10 @@ const handleChange = (editorState, line, whitelist) => {
         do {
           matchArr = re.exec(line);
           if (matchArr) {
+            if (matchArr[0][0] === " ") {
```
Comment (mxstbr) [inline src/modifiers/handleInlineStyle.js]: I think this needs to check for any whitespace char with a regex (`\s`), like the regex for inline style matchers? Using a tab or a non-breaking space will break this removal.

**Review by mxstbr — COMMENTED** [review]
This is awesome! Just two tiny nitpicks, then we're ready to go 👍

**Comment by andrewfan** [comment]
@mxstbr done

**Review by mxstbr — APPROVED** [review]
This is great, thanks so much!

**Pull request merged**